roles/terminal/tasks/ctags.yml

```diff
@@ -14,14 +14,14 @@
     with_items:
         - autoconf
         - pkg-config
-    become: true
+    become: yes
```

Comment:
and now you don't have to run sudo ansible anymore, it works correctly for users